server/settings_test.go

```diff
@@ -44,6 +44,7 @@ func TestSettings(t *testing.T) {
 							Lr: "60s",
 						},
 						DataRetention: "720h",
+						AWSPartitions: []string{"aws"},
```

Comment:
what about the case when we don't want to change `AWSPartitions`

Reply:
That particular line restores default values after other tests – that's teardown func.

Reply:
Added DefaultsAreNotRestored